include/RAJA/pattern/WorkGroup/WorkStruct.hpp

```diff
@@ -45,7 +45,7 @@ struct WorkStruct;
  *   sizeof(GenericWorkStruct) <= sizeof(WorkStruct<size>)
  */
 template < typename Dispatcher_T >
-using GenericWorkStruct = WorkStruct<alignof(std::max_align_t), Dispatcher_T>;
+using GenericWorkStruct = WorkStruct<RAJA_HOST_MAX_ALIGN, Dispatcher_T>;
```

Comment:
Could we do `#define RAJA_HOST_MAX_ALIGN alignof(std::max_align_t)` 
and is there something similar for the device?

Reply:
That's kinda what my first thought was too, if it's possible. My guess is that there probably is not something similar for a device.

I think options 2 and 3 above are too rigid. I don't know enough about the WorkGroup stuff to say whether 1 is a good option. Is there some way this could be conveniently handled via something like an execution policy type?

Reply:
Something we could do is to use 16 directly, and add a static_assert that 16 >= alignof(std::max_align_t).